lib/eslint-patch.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+var meld = require('meld');
+var docs = require('./docs');
+
+var supportedPlugins = ['react', 'babel'];
+
+module.exports = function patcher(eslint) {
+
+  meld.around(eslint.CLIEngine, 'loadPlugins', function(joinPoint) {
```

Comment:
Is this patch working as intended? I arrived here because I'm seeing `Failed to load plugin` errors for an unsupported plugin, but I _think_ this patch is intended to avoid loading unsupported plugins?

I'm questioning whether this does anything because it appears to me that `eslint.CLIEngine` has not had a `loadPlugins` method since [this commit](https://github.com/eslint/eslint/commit/b28a19d1b708e42bf54c0afc6df363b2c3d250a3), and that commit has been around since ESLint 2.0.0?